style.css

```diff
@@ -0,0 +1,566 @@
+* {
+  margin: 0;
+  padding: 0;
+  font-family: "Jua", sans-serif;
+}
+
+.transform {
+  transform: rotate(-11deg);
+  box-shadow: 10px 20px 10px -15px black;
+}
+
+header,
+html {
+  width: 100%;
+}
+
+h2 {
+  display: none;
+}
+
+footer div {
+  margin: 0 40px 0 40px;
+}
+
+footer h4 {
+  font-size: 20px;
+  color: #fff;
+  padding-top: 35px;
+  margin-bottom: 35px;
+}
+
+footer > div > ul > a {
+  text-decoration: none;
+  color: #000;
+}
+
+nav {
+  background-color: gray;
+  width: 100%;
+  height: 175px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-bottom: 15px;
+}
+
+.logo {
+  background-color: #696969;
+  width: 210px;
+  height: 76px;
+  margin: 5px 15px;
+  position: relative;
+}
+
+.navbar {
+  text-align: center;
+}
+
+.navbar > a {
+  margin: 0 auto;
+  color: rgb(255, 248, 248);
+  font-size: 16px;
+  text-decoration: none;
+  font-weight: 600;
+}
+
+.navicon {
+  background-color: rgb(102, 101, 101);
+  margin: 0 auto;
+  height: 49px;
+  width: 35px;
+}
+
+.search {
+  position: relative;
+  right: 30px;
+}
+
+.search > a {
+  display: flex;
+  justify-content: space-between;
+  width: 70px;
+  height: 18px;
+  border: solid white 2px;
+  padding: 10px 20px;
+  border-radius: 5px;
+}
+
+.featured-articles {
+  position: relative;
+  display: flex;
+  flex-wrap: wrap;
+  min-height: 0;
+}
+
+.latest {
+  position: absolute;
+  left: 30%;
+  width: 35%;
+  top: 2%;
+  text-align: center;
+}
+
+.latest > p {
+  background-color: rgb(255, 255, 255);
+  font-size: 40px;
+  font-weight: 600;
+  margin: 0 auto;
+  padding: 8px;
+  width: fit-content;
+}
+
+.seriously {
+  position: absolute;
+  display: flex;
+  color: rgb(255, 255, 255);
+  width: 230px;
+  top: 20px;
+  right: 20px;
+}
+
+.seriously p {
+  border: solid white 1px;
+  padding: 2px 5px;
+  margin-left: 10px;
+  border-radius: 5px;
+}
+
+svg {
+  fill: rgb(143, 137, 137);
+  height: 60px;
+  width: 60px;
+  background-color: rgb(255, 248, 248);
+}
+
+.intro {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  grid-template-rows: repeat(2, 1fr);
+}
+
+.intro p {
+  font-size: 45px;
+  font-weight: 600;
+}
+
+.article1 {
+  background-color: rgb(160, 159, 159);
+}
+
+.article2 {
+  background-color: rgb(116, 114, 114);
+}
+
+.article3 {
+  background-color: rgb(155, 155, 155);
+}
+
+.article4 {
+  background-color: rgb(80, 75, 75);
+}
+
+.latest-article {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  margin-top: 10px;
+  grid-template-rows: repeat(3, 1fr);
+  grid-row-gap: 10px;
+  grid-column-gap: 10px;
+  background-color: #f6f3f2;
+}
+
+.photo-name {
+  display: flex;
+  margin: 200px 100px;
+}
+
+.photo-name p {
+  font-size: 20px !important;
```

Comment:
- [ ] Don't use !important in your style. [Read more about specificity in CSS here](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity) for alternatives if you are having issues applying this rule.